app/routes/transcription_ws.py

```
@@ -1,5 +1,27 @@
+Dependencies:
+- fastapi: For creating the FastAPI application and handling WebSocket connections.
+- app.services.transcription.transcriber: For transcribing base64 audio data.
+- loguru: For logging information about the WebSocket connection and any exceptions that occur.
+
+Authors: @kcaparas1630
+        @William226
+
+"""
 from fastapi import APIRouter, WebSocket, WebSocketDisconnect
-from app.services.transcription.transcriber import transcribe_base64_audio
+from ..services.transcription.transcriber import transcribe_base64_audio
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)